pkgs/development/compilers/go/1.22.nix

```diff
@@ -179,7 +179,7 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   meta = with lib; {
-    changelog = "https://go.dev/doc/devel/release#go${lib.versions.majorMinor finalAttrs.version}";
+    changelog = "https://go.dev/doc/devel/release#go${finalAttrs.version}";
```

Comment:
I like this change but it shouldn't be in the update commit. Can you please move it into its own commit? Thanks!